src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -1,9 +1,9 @@
-﻿using System;
+using System;
```

Comment:
@hhyyrylainen this file has lost the BOM but CI is not complaining.

Reply:
I noticed that, and a fix for the CI not actually running on the right files is on my branch (it's waiting for the feature to be done before merging): https://github.com/Revolutionary-Games/Thrive/pull/2439

Reply:
@Kasterisk
you can add the BOM to the file like [that on unix](https://stackoverflow.com/a/35807131) or like [that on windows with np++](https://stackoverflow.com/a/8432691)